CMakeLists.txt

```diff
@@ -20,6 +20,10 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 if(NOT UNIX)
     message(FATAL_ERROR "Only Unix-like systems are supported.")
 endif()
+#Language bindings
+set(GEN_LANGUAGE_BINDINGS "Enable python bindings." ON) 
+set(GEN_PYTHON_BINDINGS "Enable python bindings." ON) 
+set(GEN_PYTHON_VERSION "3" CACHE STRING "Python version")
```

Comment:
I believe Python 2 will no longer be supported, right @rkrejci ? So this option and any references should be removed.

Reply:
So we can keep it for now, I guess. But the hand-written bindings will be only for Python 3.